Docs/doc.html

```diff
@@ -0,0 +1,91 @@
+        <a class ='sidebar-item hover-effect' href="">Text Utilities</a>
+        <a class ='sidebar-item hover-effect' href="">Toasts</a>
+
+    </aside>
+    <div class="main-content">
+        <h1 class="section-header">Introduction</h1> 
+        <p>Ozone-UI is a store-house of readymade css components for the UI designs. </p>
+        <h3 class="section-sub-header">Installation</h3>
+        <p>To start using Ozone-UI 
+            you just need to import below style sheet in your project</p>
+        <iframe
+            src="https://carbon.now.sh/embed?bg=rgba%28250%2C252%2C254%2C1%29&t=night-owl&wt=none&l=auto&width=967.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14.5px&lh=152%25&si=false&es=2x&wm=false&code=%2540import%2520url%28%2522https%253A%252F%252Fozone-ui-component-library.netlify.app%252Fstyle.css%2522%29"
+            style="width: 100%; height: 96px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+        </iframe>
+
+        <h3 class="section-sub-header margin-top">Colours</h3>
+        <p>Ozone-UI comes with pre-defined colors, which are consistent across the design.
+             </p>
+        <div class="element-display">
+            <div class="color primary">Primary</div>
+            <div class="color secondary">Secondary</div>
+            <div class="color info">Info</div>
+            <div class="color success">Success</div>
+            <div class="color warning">Warning</div>
+            <div class="color danger">Danger</div>
+            <div class="color underlined">Underlined</div>
+          
+        </div>
+
+        <h3 class="section-sub-header margin-top">Typography</h3>
+        <p>Ozone-UI uses Times New Roman as default font.
+             </p>
+             <iframe
+                src="https://carbon.now.sh/embed?bg=rgba%28250%2C252%2C254%2C1%29&t=night-owl&wt=none&l=auto&width=967.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14.5px&lh=152%25&si=false&es=2x&wm=false&code=%27Times%2520New%2520Roman%27%252C%2520Times%252C%2520serif"
+                style="width: 100%; height: 96px; border:0; transform: scale(1); overflow:hidden;"
+                sandbox="allow-scripts allow-same-origin">
+           </iframe>
+
+        <div class="next-links">
+            <a class="next-link-item" href="alert.html">
+                <span class="margin-lr"><ion-icon name="arrow-undo-sharp"></ion-icon></span>Home
+            </a>
+            <a class="next-link-item" href="">Alert
+                <span class="margin-lr" ><ion-icon name="arrow-redo-sharp"></ion-icon></span> </a>
+        </div>
+       
+    </div>
+
+</main>
+    
+</body>
+<script type="module" src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+<script nomodule src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
```

Comment:
Script tags need to be inside body. [Check here](https://hackinbits.com/interview-questions/html/why-script-tags-should-be-placed-at-the-end-of-body-tag)

Reply:
okay

Reply:
Changes done.